GA-worksheet.md

```diff
@@ -26,44 +31,58 @@
 ##Project #2 Part #2: Style the Header with CSS
 
 1. What advantages does the `<link>` tag have over the `<style>` tag?
+    You can its better in an orginization sense.
 
 2. What is normalize.css?
+    The CSS File
```

Comment:
It's a special type of CSS file and serves a particular purpose...what is that purpose?